daprdocs/content/en/developing-applications/ides/visual-studio-code/vscode-dapr-extension.md

```diff
@@ -0,0 +1,77 @@
+---
+type: docs
+title: "Dapr Visual Studio Code extension overview"
+linkTitle: "Dapr extension"
+weight: 10000
+description:  "How to develop and run Dapr applications with the Dapr extension"
+---
+
+
+Dapr offers a *preview* [Dapr Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-dapr) for local development and debugging of your Dapr applications.
+
```

Comment:
It would be good to have a description in the intro of what the VSCode Extension is useful for, its not just debugging.

Reply:
Sounds good I will change it to fully recognize its usage, and will add that it does support all languages that support Dapr as it doesn't have any language specific restrictions based on my testing.